src/anyio/pytest_plugin.py

```diff
@@ -1,16 +1,22 @@
 from __future__ import annotations
 
+import sys
 from collections.abc import Iterator
 from contextlib import ExitStack, contextmanager
 from inspect import isasyncgenfunction, iscoroutinefunction
 from typing import Any, Dict, Tuple, cast
 
 import pytest
 import sniffio
+from _pytest.outcomes import Exit
```

Comment:
It's publicly available at pytest.exit.Exception

Reply:
`pytest.exit` is a function, not a module.

Reply:
Oh, I misunderstood. The exception is somehow attached to the function?

Reply:
I know it would work like that, but PyCharm currently sees this as an error:
![image](https://github.com/user-attachments/assets/8aa7817c-9565-4f69-b8e0-ffaa6cd4ead1)
I would prefer to not make this change so long as this is the case. I know it's just one of the myriad of PyCharm's bugs, but still...

Reply:
The type annotation is off the charts https://github.com/pytest-dev/pytest/blob/9a444d113658be6ccb8dc9f57ed118a1ef17c94c/src/_pytest/outcomes.py#L105